src/services/groups/service/membership.ts

```diff
@@ -31,6 +35,33 @@ export const act = function (
 } satisfies ServiceMethodDynamicSegments<
   OpenAPI.operations['update_group_v2_groups__group_id__put']['parameters']['path']['group_id'],
   {
+    query?: never;
     payload: OpenAPI.operations['group_membership_post_actions_v2_groups__group_id__post']['requestBody']['content']['application/json'];
   }
 >;
+
+/**
+ * Get the status counts of memberships for each group you are an admin or manager of.
+ * @see https://groups.api.globus.org/redoc#tag/groups/operation/get_statuses_v2_groups_statuses_get
+ */
+export const getStatuses = function (
```

Comment:
I think this method makes more sense living at `groups.groups.getStatuses()` (similar to `getMyGroups`) – the `membership.act()` is kinda weird one-off RPC, IMO.

Reply:
Yeah, I wasn't sure where to put it. You're not really getting information about the "group" so much as the "memberships", and it seemed like the `GET` to the pseudo-`PUT` of act.

I'll move it over.